javatests/arcs/android/storage/service/StorageServiceEndToEndTest.kt

```diff
@@ -173,6 +235,7 @@ class StorageServiceEndToEndTest {
                 CollectionType(EntityType(DummyEntity.SCHEMA)),
                 DummyEntity
             ),
-            storageKey
+            storageKey,
+            Ttl.Hours(1)
```

Comment:
If TTL only matters for the TTL tests, maybe we should have a separate named entity for that?

Reply:
I added it as an optional parameter. Note: this is the handle, not the entity.